frontend/src/Store/Actions/seasonPassActions.js

```diff
@@ -25,7 +26,26 @@ export const defaultState = {
   secondarySortDirection: sortDirections.ASCENDING,
   selectedFilterKey: 'all',
   filters,
-  filterPredicates,
+  filterPredicates: {
+    ...filterPredicates,
+
+    episodeProgress: function(item, filterValue, type) {
```

Comment:
This should be added to `seriesActions.filterPredicates` and removed from the 3 action files.

Reply:
Done